### ISSUE TITLE
Consider renaming the link to the blog on AMO

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-frontend#10941]
The text of the current link to the blog on AMO, which is in the footer, is "Firefox Add-on Reviews". There is some consensus on the team that this text is not very descriptive of the fact that it points to our blog. This issue is about discussing what might be better text for the link which would encourage more users to visit the blog, and to then implement the new link text. We might want to consult with Content Design on this.

**Comment by bobsilverberg** [comment]
Here's the link in context:

![Screen Shot 2021-10-04 at 2 35 26 PM](https://user-images.githubusercontent.com/142755/135905731-b2f641e4-2ffa-4946-b6e7-5dfb70682ea9.png)

**Comment by MeridelW** [comment]
Definitely agree with a new name for the footer here, and I recommend pulling that name through to the blog itself (I assume that may require some uplift/template change). 

As editor, I defer to Scott D. for final call here. Below are some options. Some considerations:
- We don't want this to sound like another version of  our "recommended extensions" category, so making it clear in the name that this is a blog or editorial/from staff would help.  
- I am a proponent of calling things what they are (this is a blog, and it is by Firefox). Looking at a site like Etsy, they have something similar and it's simply called: Etsy blog.


**Options:**
- Firefox Extensions Blog (I assume we mostly talk about extensions, not themes, so maybe you can get away with just using the more common/specific 'extensions' nomenclature here. Stylistically nice to avoid the hyphenated "add-ons" word.)
- Firefox Add-ons Blog
- Firefox featured add-ons ("featured" is close to "recommended" but a "feature" is more editorial so maybe this is okay? Don't use if "featured" is already a category of extensions on the site)
- Firefox editor picks
- Editor's choice

@devaneymoz

**Comment by devaneymoz** [comment]
I agree with Meridel's points on naming conventions and concur it might be best to simply go with a direct title like **Firefox Add-ons Blog** (we may indeed editorialize themes some day so probably best to keep it "add-ons"). Thanks!
